package.json

```diff
@@ -51,7 +51,7 @@
   ],
   "scripts": {
     "lint": "eslint src/*.js",
-    "dev-prod": "comunica-compile-config config/config-default.json > .tmp-comunica-engine.js && ./bin/queries-to-json.js && webpack serve --config webpack.config.js --mode production",
+    "dev-prod": "comunica-compile-config config/config-default.json > .tmp-comunica-engine.js && ./bin/queries-to-json.js && webpack serve --config webpack.config.dev.js --mode production",
```

Comment:
Are you sure this is correct. AFAICS, `webpack.config.dev.js` does not exist?

Reply:
Forgot to commit my change, fixed now!